pkgs/tools/misc/bat/default.nix

```
@@ -38,10 +31,51 @@ rustPlatform.buildRustPackage rec {
       --prefix PATH : "${stdenv.lib.makeBinPath [ less ]}"
   '';
 
+  passthru = {
+    tests = { inherit (nixosTests) bat; };
+
+    updateScript = writeScript "update.sh" ''
+      #!${stdenv.shell}
```

If you use `writeShellScript` instead you can avoid the shebang line and Nix will syntax-check it for you.

```suggestion
    updateScript = writeShellScript "update.sh" ''
```